source/draw/Image.ooc

```diff
@@ -20,9 +20,6 @@ Image: abstract class {
 	init: func (=_size) {
 		this _referenceCount = ReferenceCounter new(this)
 	}
-	init: func ~fromImage (original: This) {
-		this init(original size)
-	}
```

Comment:
This is all that remained after removing crop and wrap from Image so now it has no purpose.